src/main/java/clickup/requests/SpaceRequest.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2021 JalaSoft.
+ * This software is the confidential and proprietary information of JalaSoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with JalaSoft
+ *
+ * @author Jorge Caceres
+ */
+
+package clickup.requests;
+
+import clickup.ApiEndpoints;
+import clickup.entities.Space;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+public class SpaceRequest extends BaseRequest {
+
+    /**
+     * Creates a space and returns its identifier.
+     *
+     * @return a space_id
+     * @throws JsonProcessingException when the response is not a valid json
+     * @author Jorge Caceres
+     */
+    public String createSpace() throws JsonProcessingException {
+        Space space = new Space();
+        space.setName("Space created RunTest before From API");
```

Comment:
The space should also use a random name to avoid duplication in case of creating more than one with this method.